unit-tests/IRTest.hs

```diff
@@ -0,0 +1,28 @@
+module Main where
+
+import IR.IR
+import Common.Identifiers
+import Data.List
+
+{-
+How to run:
+in src, run `stack ghci`
+then do ``:l unit-tests/IRTest.hs`
+then do ``:main`
+
+-}
+tests  = [(show(New),"new"), (show (Var (VarId (Identifier "foo")) ()),"(var foo () )")]

```

Comment:
This is fine for now, but we should really set up hspec for unit and regression tests